content/docs/5.1/develop/rawkv/put-get-delete.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Put Get Delete
+description: How to use RawKV's basic operations such as Put, Get, and Delete.
+menu:
+    "5.1":
+        parent: RawKV
+        weight: 1
+---
+
+This document walks you through how to use RawKV's basic operations such as `Put`, `Get` and `Delete`.
+
+## Java
+
+### Step 1: import 
+
+First of all, import all the packages used in the example.
+
+```java
+import java.util.Optional;
+import org.tikv.shade.com.google.protobuf.ByteString;
+import org.tikv.common.TiConfiguration;
+import org.tikv.common.TiSession;
+import org.tikv.raw.RawKVClient;
+```
+
+`com.google.protobuf.ByteString` is used as the type of Key and Value. 
+
+To avoid conflict, `com.google.protobuf.ByteString` is shaded to `org.tikv.shade.com.google.protobuf.ByteString` and is included in the client package.
```

Comment:
do you mean `shared`?

Reply:
https://maven.apache.org/plugins/maven-shade-plugin/

Reply:
`shared` is used to rename the package name.